### ISSUE TITLE
[BUG] Auto-addition of study during arc a add

### ISSUE THREAD

**Issue by Brilator** [nfdi4plants/arcCommander#49]
**Describe the bug**
I can follow the rationale, but I'm a bit unhappy, with the handling of the study layer. e.g.
> If no study identifier is given, a study is created with the assay identifier as study identifier.
This might become confusing at some point. 

a) There is no isa.study.xlsx added during `arc a add`. So the ARC <-> isa.investigation.xlsx becomes inconsistent. 
b) If I add an assay without associating a study and then choose to remove the assay, the study persists in the isa.investigation.xlsx. So I would have to remove something I have not actively added. 

**To Reproduce**
1. `arc a add --assayidentifier SomeTestAssayWithoutStudy`
2. Check folder to find no "SomeTestAssayWithoutStudy.study.xlsx"
3. `arc a remove --assayidentifier SomeTestAssayWithoutStudy `
4. See isa.investigation.xlsx: study still exists. 

**Expected behavior**
a) Add study.xlsx when out-adding study during assay creation
b) Remove study that was solely added for one assay. 

**OS and framework information (please complete the following information):**
 - OS: MacOS Big Sur
 - OS Version 11.3.1

**Additional context**
Add any other context about the problem here, like ideas for a solution.

**Comment by muehlhaus** [comment]
Thank you for the valuable report. You are absolutely right. When adding an assay a study including a corresponding ISA file needs to be created. 
I am not sure about the removal so. Once one created a study implicit or explicit, the removal should be explicit by the ‘remove study’ command. 
We should include both scenarios into the ARC specification:
-	No assay without study. 
-	Are empty studies without assays allowed?

**Comment by Brilator** [comment]
Agreed, the removal is not as clear. Maybe add a message during `arc a add` like "no study identifier supplied, auto generated _studyXY_ for you". 

I would definitely allow empty studies without assays, although I cannot fully see yet, how much the study layer is going to be used. My vision is, that users would initiate a collaboration (e.g. between groups) by sharing an (assay-)empty ARC with only investigation + study file(s). So before the actual experiments are done, design them from scratch in an ARC.

**Comment by HLWeil** [comment]
Okay so after #52 the current state is as follows: 

- The arc add/register command is more verbose about the study being added and also the naming of this study

- Studies created this way are **not** removed when the assay gets removed

If this matches your expectations, feel free to close this issue.

**Comment by Brilator** [comment]
Good. But this isn't in production yet, right? Or at least during `arc a add -a SomeAssay` the SomeAssay.study.xlsx is not added.

**Comment by HLWeil** [comment]
Yes, it's merged into the developer branch. But no package was released yet afterwards.
